docs/content/intro-con.adoc

```diff
@@ -1,25 +1,38 @@
 [id='concept-explanation-{context}']
 = Overview of {PRODUCT}
 
-
 Red Hat Managed Integration delivers managed integration services hosted on the leading container platform, enabling diverse developers to engage in agile, microservices-based application development.
 {PRODUCT_SHORT} enables your team to develop, build and deploy enterprise integration solutions on an OpenShift Dedicated cluster managed by Red Hat.
 {PRODUCT_SHORT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components.
 {PRODUCT_SHORT} enables the operations team focus on high value objectives while Red Hat manage the infrastructure. All the applications and services are protected by Red Hat Single Sign-On.
 
-The following middleware products are available as part of {PRODUCT_SHORT}:
+== Solution Explorer
+The Solution Explorer is a web application that serves as a user friendly gateway to an {PRODUCT_SHORT} cluster.  It contains links to middleware products that are already installed and ready to use with no configuration necessary, with access to each product managed through RHSSO.
```

Comment:
user-friendly gateway... seems colloquial
contains links - might translate badly
product might be misleading

Reply:
Fixed after conversation with Paul.